### PR TITLE
Cross line command

### PR DESCRIPTION
**Pull request by gowabash**
Simple command to drive until the robot is within 20 inches of an obstacle.  Also includes configuration and calibration of the ultrasonic sensor

### CODE REVIEW

File: src/org/usfirst/frc/team1024/robot/Robot.java [hunk 1]
```diff
@@ -39,6 +43,8 @@ public void robotInit() {
 		m_chooser.addDefault("Default Auto", new Auto());
```
Comment (MrDoctorb) [inline src/org/usfirst/frc/team1024/robot/Robot.java:43]: What is m_chooser?

File: src/org/usfirst/frc/team1024/robot/commands/CrossLine.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package org.usfirst.frc.team1024.robot.commands;
```
Comment (MrDoctorb) [inline src/org/usfirst/frc/team1024/robot/commands/CrossLine.java:1]: Should this be run from the Auto Class?

**Review by MrDoctorb — COMMENTED** [review]
Good code. I think the auto should be run from the Auto class even if it's still a test. Have the cross line class be run by running the Auto class.

File: src/org/usfirst/frc/team1024/robot/Robot.java (hunk 1 above)
Comment (briforge) [reply src/org/usfirst/frc/team1024/robot/Robot.java:43]: It's declared above as a SendableChooser<Command>

File: src/org/usfirst/frc/team1024/robot/commands/Auto.java [hunk 3]
```diff
@@ -24,5 +26,6 @@ public Auto() {
         // e.g. if Command1 requires chassis, and Command2 requires arm,
         // a CommandGroup containing them would require both the chassis and the
         // arm.
+
```
Comment (briforge) [inline src/org/usfirst/frc/team1024/robot/commands/Auto.java]: If this Auto CommandGroup doesn't do anything, then either delete it or remove it from this pull request.

Comment (briforge) [reply src/org/usfirst/frc/team1024/robot/commands/Auto.java]: or undo the changes (just in this file), and re-commit it, then it won't show up here as a changed file.

**Review by briforge — APPROVED** [review]
good, ready to merge

**Pull request merged**